saleor/dashboard/order/forms.py

```diff
@@ -152,25 +151,25 @@ def __init__(self, *args, **kwargs):
     def clean_quantity(self):
         quantity = self.cleaned_data['quantity']
         delta = quantity - self.initial_quantity
-        stock = self.instance.stock
-        if stock and delta > stock.quantity_available:
+        variant = self.instance.variant
```

Comment:
I'm not sure why instead of putting a column ["product" that is referencing the table "Product"](https://github.com/mirumee/saleor/blob/master/saleor/order/models.py#L160-L164), we don't put a column "variant" referencing "ProductVariant" (also see https://github.com/mirumee/saleor/blob/master/saleor/order/utils.py#L124-L127, where we are creating order lines from variants).

Reply:
It will be done in this PR as well, just taking it down one by one